tools/FindPythonLibsNew.cmake

```diff
@@ -1,84 +1,74 @@
-# - Find python libraries
-# This module finds the libraries corresponding to the Python interpreter
-# FindPythonInterp provides.
-# This code sets the following variables:
+# * Find python libraries This module finds the libraries corresponding to the Python interpreter
```

Comment:
just to clarify: only whitespace changes in this file?

Reply:
Yes, I didn't touch this.